src/remote-backend/remote-backend.hh

```diff
@@ -301,6 +327,36 @@ protected:
   void logRequest(const Pistache::Rest::Request &request);
   void logRequestResponse(const Pistache::Rest::Request &request, const Pistache::Http::Response &response, const nlohmann::json& ret);
 
+  struct FedRecords {
```

Comment:
```suggestion
  struct FedRecord {
```

or maybe even leave out `Fed` because this type does not actually care that is being used in `feedRecords`